NEWS.md

```diff
@@ -124,6 +124,32 @@ The format is based on [Keep a Changelog]
     
 ### Changed
 
+- **Breaking change**: When a Wayland-native window is snapped to a screen
```

Comment:
No need for `**Breaking change**`. By definition, anything in the change section refers to `changes in existing functionality`.

Reply:
Maybe we could move the `Changed` to the top? At least in my case I usually don't read through all the detailed changes on updates but that section would be important to see.

Also renaming the whole section to `Breaking changes` (or something similar) might help as well to differentiate the importance of the sections.

Reply:
I moved the "Changes" section up top, called it "Breaking Changes", and also reworded the discussion about `app_id` for good measure.

Reply:
> Maybe we could move the `Changed` section to the top? At least in my case I usually don't read through all the detailed changes on updates but that section would be important to see.

I prefer to keep the order it as it was. We've been sticking with https://keepachangelog.com/en/1.0.0/ for a couple of years, and I think the consistent pattern is important.

We have a `Changed` section for pretty much every release and the entries within these all describe changes to functionality which are "breaking" to to some degree.

> Also renaming the whole section to `Breaking changes` (or something similar) might help as well to differentiate the importance of the sections.

How about adding a note at the top about the `### Changed` label explaining that it refers to breaking changes and that we go to great lengths to avoid these, but if there is a strong argument in aid of consistency, code correctness or maintainability then we accept them - particularly if the breaking change has a low impact or small footprint.
Maybe we should also describe that whilst `labwc` is not a library, it is a platform upon which people build other things,  and that we value stability and a consistent interfaces.

Problems with changing to `Breaking Changes` include (a) that `Added`, `Changed`, etc are verbs in past tense so it breaks the pattern and I can't think of a sensible way of including `Breaking` with a verb. Also it breaks the keepachangelog standard.

Reply:
Ok, back to the original order and heading, but I kept the rewrites from the last revision.

I don't have a strong opinion about some disclaimer text at the top. While it's important to point out behavioral changes that might intrude on somebody's workflow, a statement about preferring to keep stable interfaces probably just invites *post hoc* criticism from people who disagree with arguments for improvement of some feature or another and would prefer old behavior.

Reply:
> a statement about preferring to keep stable interfaces probably just invites post hoc criticism from people who disagree with arguments for improvement of some feature or another and would prefer old behavior

Yes, good point. Let's leave as you've done it.